server/services/assessRisksAndNeedsService.test.ts

```diff
@@ -40,14 +40,20 @@ describe(AssessRisksAndNeedsService, () => {
     })
 
     it('provides a userMessage on failure', async () => {
-      restClientMock.get.mockRejectedValue(createError(404))
+      restClientMock.get.mockRejectedValue(createError(409))
       try {
         await assessRisksAndNeedsService.getRiskSummary('crn123', 'token')
       } catch (err) {
-        expect(err.status).toBe(404)
-        expect(err.userMessage).toBe('Could not get service user risk scores from OASys.')
+        expect(err.status).toBe(409)
+        expect(err.userMessage).toBe("Could not get service user's risk scores from OASys.")
       }
     })
+
+    it('return null when encountering 404 errors', async () => {
```

Comment:
What do you think about describing what 404 means in the test name?

Something similar to
```suggestion
    it('return null when the risk information does not exist', async () => {
```

Reply:
done